client/lib/products-values/index.js

```diff
@@ -152,6 +153,13 @@ export function isBusiness( product ) {
 	return isBusinessPlan( product.product_slug );
 }
 
+export function isEcommerce( product ) {
```

Comment:
Should this be added as an additional check to `isPlan()`?
https://github.com/Automattic/wp-calypso/blob/faaebbeed6e6b79157a9ee82a60d867dacee2e70/client/lib/products-values/index.js#L230

Reply:
Thanks, great catch! I have added that in bba959ea6e.